src/ui/components/shared/SettingsModal/SettingsBody.tsx

```diff
@@ -145,61 +37,36 @@ export function SettingsBodyHeader({ children }: { children: React.ReactChild })
   return <h2 className="text-2xl">{children}</h2>;
 }
 
-export default function SettingsBody({ selectedSetting, userSettings }: SettingsBodyProps) {
-  const { title, items } = selectedSetting;
+export default function SettingsBody<
+  T extends string,
+  V extends Record<string, unknown>,
+  P extends Record<string, unknown>
+>({ onChange, panelProps, selectedSetting, values }: SettingsBodyProps<T, V, P>) {
+  const { title } = selectedSetting;
   const [showRefresh, setShowRefresh] = useState(false);
```

Comment:
Is the `showRefresh` mechanism being used anywhere? If not, we can probably take it out.

Reply:
Doesn't look like it. I've cleaned it up!